app/main.cpp

```diff
@@ -1,7 +1,25 @@
 #include <iostream>
 #include <format>
 #include "app.h"
-
+#include<iomanip>
+using vsite::oop::v2::color;
 int main()
 {
+	double r, g, b;
+	std::cin >> r;
+	std::cin >> g;
+	std::cin >> b;
+	color c;
+	
+	c.set_red(r);
+	c.set_green(g);
+	c.set_blue(b);
+
+	std::cout << "Crvena boja: " << c.get_red() << "\n";
+	std::cout << "Zelena boja: " << c.get_green() << "\n";
+	std::cout << "Plava boja: " << c.get_blue() << "\n";
+
+	std::cout << '#' << std::setfill('0') << std::setw(6) << vsite::oop::v2::to_hex(r) + vsite::oop::v2::to_hex(g) + vsite::oop::v2::to_hex(b);
```

Comment:
Hm, `r`, `g` i `b` su `double`-ovi (između 0 i 1) sa kojima zovete `to_hex` (koja prima `int`)? Što to radi uopće?
Ideja je da ispišete ono što vrati `get_color_ref()`. I onda vam neće trebati ni `setfill` ni `setw` jer to sve riješite sa `std::format` (unutar `to_hex`).